astacus/coordinator/plugins/cassandra/restore_steps.py

```diff
@@ -26,11 +28,21 @@
 logger = logging.getLogger(__name__)
 
 
+def _create_network_topology_strategy_cql(keyspace: CassandraKeyspace) -> str:
+    new_replication_strategy = cm.NetworkTopologyStrategy(keyspace.network_topology_strategy_dcs)
+    replication = new_replication_strategy.export_for_schema()
+    return f"CREATE KEYSPACE {keyspace.name} WITH replication = {replication} AND durable_writes = {keyspace.durable_writes}"
```

Comment:
Should we escape the keyspace name (because of how case is handled by Cassandra ?)

Reply:
actually, I've got a better idea - let's ask the driver to do the magic for us